paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -89,6 +90,23 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
         }
     }
 
+    @SuppressLint("MissingSuperCall")
+    override fun onNewIntent(intent: Intent?) {
```

Comment:
I left this in another comment. But we probably want tests for what happens here if payment sheet wasn't already launched.

Reply:
This has been removed. I'll add tests for ExternalPaymentMethodActivity, which now handles this